src/features/MoviesBrowser/PopularMovies/TileMovie/index.js

```diff
@@ -8,21 +8,21 @@ import {
   NumberOfRating,
   AverageRating,
   StyleStarIcon,
+  Links,
 } from "./styled";
 import { GenresList } from "../../MoviesGenre/index.js";
 
 export const TileMovie = ({ repositories }) => {
   const movies = Object.values(repositories.results);
-  console.log(movies)
 
   return (
       <>
           {movies.map(({id, title, release_date, poster_path, vote_count, vote_average, genre_ids }) => (
 
               <Container key={id}>
                   <Image src={`https://image.tmdb.org/t/p/original/${poster_path}`} alt="" />
-                  <Description>
-                      <DescriptionTitle>{title}</DescriptionTitle>
+                      <Description>
+                      <DescriptionTitle><Links to={`/movies/${id}`}>{title}</Links></DescriptionTitle>
                       <Data>{release_date}</Data>
                       <GenresList genre_ids={genre_ids}/>
                       <Rating>
```

Comment:
here, You can use already created Rating component.
Rating has variants; you can add your variant if it's necessary,  we can use one component in a few places ;)

Reply:
I thought about that too, but I'll deal with it in the next step.

Reply:
Great! Less means more 😅